src/com/article-changer-v3.js

```diff
@@ -82,12 +82,13 @@
 		var isLoading = false;
 		var isAnimating = false;
 		var loadingUrl = '';
+		var nextPage;
 
 		var checkStartAnimAnd = function (current, next, o) {
 			isAnimating = false;
 			if (!isLoading) {
 				//Complete anim
-				o.endAnimToArticle(current, next, o);
+				o.endAnimToArticle(current, !!next && next.length ? next : nextPage, o);
```

Comment:
This looks scary to me. Really ? We can't avoid this ?

Reply:
@PascalPiche  ??

Reply:
Nope. scope problem. in some case next will be null.